tests/testUDPSender.py

```diff
@@ -10,6 +10,9 @@
 """
 
 from pyniexp.connection import Udp
+import keyboard
```

Comment:
Is "keyboard" package specified in requirements?

Reply:
It is specified in the requirements of pyniexp.